src/ServicePulse.Host/vue/src/stores/HeartbeatsStore.ts

```diff
@@ -107,6 +109,13 @@ export const useHeartbeatsStore = defineStore("HeartbeatsStore", () => {
     filterString.value = filter;
   }
 
+  async function deleteEndpoint(endpoint: Endpoint) {
+    useShowToast(TYPE.INFO, "Info", "Removing Endpoint");
```

Comment:
note: legacy implementation didn't have the toast notifications, but if the delete operation was stalled for some reason (e.g. DB lock) then the user wouldn't have any feedback

Reply:
Should this use `autoRefresh.executeAndResetTimer()`? otherwise the delete call may happen, update the UI, then an auto-refresh call brings it back, then a subsequent removes it again.